docs/CHANGELOG.asciidoc

```diff
@@ -52,6 +53,235 @@ Review important information about the {kib} 8.x releases.
+* Adds service profiling flamegraph ({kibana-pull}165360[#165360]).
+Cases::
+* Adds custom fields in Cases ({kibana-pull}167016[#167016]).
+Dashboard::
+* Copy panel refactor ({kibana-pull}166991[#166991]).
+* Make links panel available under technical preview ({kibana-pull}166896[#166896]).
+* Store view mode in local storage ({kibana-pull}166523[#166523]).
+* Adds a read only state for Managed Dashboards ({kibana-pull}166204[#166204]).
+Discover::
+* Adds resize support to the Discover field list sidebar ({kibana-pull}167066[#167066]).
+Elastic Security::
+For the Elastic Security 8.11.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+Enterprise Search::
+For the Elastic Enterprise Search 8.11.0 release information, refer to {enterprise-search-ref}/changelog.html[_Elastic Enterprise Search Documentation Release notes_].
+Fleet::
+* Set env variable `ELASTIC_NETINFO:false` in {kib} ({kibana-pull}166156[#166156]).
+* Added restart upgrade action ({kibana-pull}166154[#166154]).
+* Adds ability to set a proxy for agent binary source ({kibana-pull}164168[#164168]).
+* Adds ability to set a proxy for agent download source ({kibana-pull}164078[#164078]).
+Lens & Visualizations::
+* Adds color mapping for categorical dimensions in *Lens* available under technical preview ({kibana-pull}162389[#162389]).
+* Inline editing of **Lens** panels on a dashboard or canvas ({kibana-pull}166169[#166169]).
+* Individual annotation editing from library ({kibana-pull}163346[#163346]).
+Logs::
+* Convert log explorer profile into standalone app available under technical preview ({kibana-pull}164493[#164493]).
+Machine Learning::
+* Adds support for the ELSER v2 download in the Trained Models UI ({kibana-pull}167407[#167407]).
+* Adds data drift detection workflow from Trained Models to Data comparison view ({kibana-pull}162853[#162853]).
+Management::
+* Supports for viewing and editing data retention per data stream in Index Management is available under technical preview ({kibana-pull}167006[#167006]).
+* Supports for viewing and editing data retention per data stream in Index Management is available under technical preview ({kibana-pull}167006[#167006]).
+* Index details can now be viewed on a new index details page in Index Management ({kibana-pull}165705[#165705]).
+* Supports for managing, executing, and deleting enrich policies in Index Management ({kibana-pull}164080[#164080]).
+Platform::
+* ES|QL, a new query language, is available under technical preview in Discover and Dashboards ({kibana-pull}146971[#146971]).
+Querying & Filtering::
+* Saved queries can now be shared between multiple spaces ({kibana-pull}163436[#163436]).
+Uptime::
+* Adds a document viewer to the summary pings table ({kibana-pull}163926[#163926]).
+
+
+For more information about the features introduced in 8.11.0, refer to <<whats-new,What's new in 8.11>>.
+
+
```

Comment:
@amyjtechwriter  I notice that in the [8.11 Kibana release notes](https://www.elastic.co/guide/en/kibana/master/release-notes-8.11.0.html) the "Enhancements and bug fixes" are showing up as a separate page.

![Screenshot 2023-10-31 at 10 26 30 AM](https://github.com/elastic/kibana/assets/41695641/0a891e07-e929-4ed9-99ef-4ddce8c9eb15)

To fix that I think you need to add a `[float]` tag here.

Reply:
Oh hang on, I'm now not sure if that is by design. Looking at older major releases (8.6.0 and 8.5.0) and they display like that as well. For 8.10.0 there isn't a separate page for Enhancements and bug fixes, so maybe I've made a mistake there. 

@gchaps Do you have a preference on the toc/display here that I can follow going forward? Is it best to keep everything in one section or separate out Enhancements and bug fixes?

Reply:
'Enhance and bug fixes` should show up on a separate page.  That page should also be included in the TOC.  It currently does not.

The [8.10.0 release notes](https://www.elastic.co/guide/en/kibana/master/release-notes-8.10.0.html) also need updating. `Enhancements and bug fixes` should be on a separate page and also appear in the TOC.

8.9.0 release notes and earlier look good.